Dockerfile

```diff
@@ -0,0 +1,40 @@
+ARG GO_VERSION=1.23.1
```

Comment:
Could you split this into two separate MRs? One for docker related functionality, the other for the vapid keys file.